docs/topics/data-analysis/data-analysis-connect-to-db.md

```diff
@@ -1,20 +1,158 @@
 [//]: # (title: Connect and retrieve data from databases)
 
-Simple guide how to start working data sources with Kotlin Notebooks.
+Kotlin Notebook offers capabilities for connecting to and retrieving data from various types of SQL databases, such as 
+MariaDB, PostgreSQL, MySQL, and SQLite. 
+Utilizing the [DataFrame library](https://kotlin.github.io/dataframe/gettingstarted.html), Kotlin Notebook can establish 
+connections to databases, execute SQL queries, and import the results for further operations.
+
+For detailed example, see the [Notebook in the KotlinDataFrame SQL Examples GitHub repository](https://github.com/zaleslaw/KotlinDataFrame-SQL-Examples/blob/master/notebooks/imdb.ipynb).
 
 ## Before you start
 
-* Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
-* Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook).
-* Add a dependency to the DataFrame library (?)
+1. Download and install the latest version of [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/download/?section=mac).
+2. Install the [Kotlin Notebook plugin](https://plugins.jetbrains.com/plugin/16340-kotlin-notebook) in IntelliJ IDEA.
+
+   > Alternatively, access the Kotlin Notebook plugin from **Settings/Preferences** | **Plugins** | **Marketplace** within IntelliJ IDEA.
+   >
+   {type="tip"}
+
+3. Create a new Kotlin Notebook by selecting **File** | **New** | **Kotlin Notebook**.
+4. In the Kotlin Notebook, import the Kotlin DataFrame library by running the following command:
+
+```kotlin
+%use dataframe
+```
+
+5. Ensure you have access to an SQL database, either locally hosted, such as MariaDB or MySQL, or remotely hosted on a 
+server or cloud platform.
```

Comment:
please remove about remotely hosted or cloud platform (we could have some issues here, need to test)

Reply:
will remove - thank you!